docs/sources/administration/team-management/administering-grafana-teams.md

```diff
@@ -0,0 +1,56 @@
+---
+aliases:
+  - 
+description: Learn how to administer Grafana Teams.
+labels:
+  products:
+    - enterprise
+    - oss
+    - cloud
+keywords:
+  - authorization
+  - microservices
+  - architecture
+menuTitle: Administering Grafana Teams
+title: Administering Grafana Teams
+weight: 300
+---
+
+# Administering Grafana Teams
+
+This topic describes how to administer Grafana Teams.
+
+## View a list of Teams
+
+See the complete list of teams in your Grafana organization.
+
+To view a list of teams:
+
+- Sign in to Grafana as an organization administrator or a team administrator.
+
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. The role you use to sign in to Grafana determines how you see Teams lists. 
```

Comment:
Should this be one single ordered list rather than an unordered list and then a shorter ordered list?